strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -465,8 +463,7 @@ protected void doDeletePath(RepositoryPath repositoryPath,
     {
         Files.delete(repositoryPath);
 
-        Repository repository = repositoryPath.getFileSystem()
-                                              .getRepository();
```

Comment:
I'm not counting the lines of code at all... It's not about that. 

`for`/`do`/`while` statements should have curly braces. 

Classes and methods should have a curly brace on a new line.

I'm fine with

```
foo.bar()
   .do()
   .it();
```

What I'm not okay with is diff-s where the IDE, (or the developer), have done a "code reformat/re-ordering" on an entire class, when there are in fact ten lines of changes, and the operation has added another 50 changes. This is what I'm talking about.